engines/terraform/outputs.go

```diff
@@ -0,0 +1,22 @@
+package terraform
+
+import (
+	"github.com/aws/jsii-runtime-go"
+	"github.com/hashicorp/terraform-cdk-go/cdktf"
+)
+
+func (td *TerraformDeployment) createOutputs() {
+	for name, module := range td.terraformResources {
+		td.createOutputsForModule(name, module)
+	}
+
+	for name, module := range td.terraformInfraResources {
+		td.createOutputsForModule(name, module)
+	}
+}
+
+func (td *TerraformDeployment) createOutputsForModule(name string, module cdktf.TerraformHclModule) {
+	cdktf.NewTerraformOutput(td.stack, jsii.Sprintf("%s_outputs", name), &cdktf.TerraformOutputConfig{
```

Comment:
Currently outputs are scoped to `<resource_name>_outputs`, but this can possibly just be `resource_name`?

Reply:
I think the _outputs is redundant and most modules and examples don't use it.